private/buf/cmd/buf/command/push/push.go

```diff
@@ -346,6 +346,37 @@ func createUploadModulesIfNotExist(
 	return nil
 }
 
+func validateFlags(flags *flags) error {
+	if err := validateCreateFlags(flags); err != nil {
+		return err
+	}
+	if err := validateLabelFlags(flags); err != nil {
+		return err
+	}
+	return nil
+}
+
+func validateLabelFlags(flags *flags) error {
```

Comment:
Put below `validateCreateFlags` since this is the call order and it is more natural

Reply:
Done!